### DISCUSSION TITLE
How could I start Appium@2.0beta56 server for two difference devices

### DISCUSSION THREAD

**Discussion by justATester3939** [appium/appium#18313] [Q&A]
I have a situation, only one macbook to code and run daily automation test case.
+ Daily run script and new script in difference folder => ok
+ I'm using appium 2.0@beta56 
   - appium -p 4723: for daily script in read devices
   - appium -p 5050: for inspector and run new script in simulator

=> but it's still conflict: when I start inspector it interrupt daily script running

Please help me!

**Comment by justATester3939** [comment]
I also change IP 0.0.0.0 and 127.0.0.1
But still not working

**Comment by jlipps** [comment]
Running multiple sessions in parallel requires not just a different Appium server port. Depending on the driver you will also want to ensure that capabilities representing other ports are unique as well. [This article](https://www.headspin.io/blog/running-multiple-appium-tests-in-parallel) might help

**Reply by justATester3939** [reply]
Thanks @jlipps I am looking into it

**Comment by justATester3939** [comment]
No answer
